build.cake

```diff
@@ -74,11 +74,13 @@ Task("TestUsingExampleImplementation")
     .IsDependentOn("ReplaceStubWithExample")
     .Does(() => {
         var allProjects = GetFiles(buildDir + "/*/*.csproj");
-        Parallel.ForEach(allProjects, parallelOptions, (project) => DotNetCoreTest(project.FullPath));
+       Parallel.ForEach(refactoringProjects, (project) => {
```

Comment:
Hello @thirdppp! Could you give us some more context on what problem you're trying to solve here?

@ErikSchierboom knows the build system better than I do, but my understanding is that `DotNotCoreTest` now contains `DotNetCoreBuild` so it's not necessary to explicitly call it.

Reply:
@thirdppp , @robkeim is correct in this case. DotNetCoreTest() will call the build for us so there's no need to put it into the ForEach as we'd be building the solution twice.

Reply:
Note: this is a feature of the .NET Core version 2.0, it doesn't work with version 1.x